### PR TITLE
player stats recorded

### PR DESCRIPTION
**Pull request by ishakanodia**
done with the changes again! try this.. it is buggy still but it's somwthing

### CODE REVIEW

**Comment by justinorringer** [comment]
This game is crazy The player doesnt take damage, the bats kill themselves, and then the player just disappears haha.

File: Scenes/Player/PlayerStats.tscn [hunk 1]
```diff
@@ -3,3 +3,4 @@
 [ext_resource path="res://Scenes/Player/Stats.tscn" type="PackedScene" id=1]
 
 [node name="PlayerStats" instance=ExtResource( 1 )]
+max_health = 3
```
Comment (justinorringer) [inline Scenes/Player/PlayerStats.tscn:6]: You shouldnt need a PlayerStats scene. The player stats persist in game_state. So at the beginning of player script you can call var game_state = get_node("/root/GameState"). This has set_heath, set_max_health functions

Comment (josejulio) [reply Scenes/Player/PlayerStats.tscn:6]: You dont need to fetch the `GameState` it's a global variable :D

Comment (ishakanodia) [reply Scenes/Player/PlayerStats.tscn:6]: I was trying to do it without the playerstats.. wasnt working well
Can try again - maybe i did something wrong along the way

and yup one can just call GameState, that's what I had started trying

**Comment by josejulio** [comment]
Tweaked in #124

**Pull request closed without merging**